packages/edit-site/src/components/sidebar-navigation-screen-template/style.scss

```diff
@@ -3,27 +3,27 @@
 	align-items: center;
 	justify-content: space-between;
 	margin-top: $grid-unit-30;
+}
 
-	&-author {
-		display: inline-flex;
-		align-items: center;
-
-		img {
-			border-radius: $grid-unit-15;
-		}
+.edit-site-sidebar-navigation-screen-template__added-by-description-author {
+	display: inline-flex;
+	align-items: center;
```

Comment:
These are minor refactorings. In my experience, splitting selectors with ampersand is not recommended and is almost never used in the codebase.